cluster/saltbase/salt/kubelet/default

```diff
@@ -116,5 +116,10 @@
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% endif -%}
 
+{% set kubelet_port = "--port=10250" -%}
```

Comment:
why is this set to "--port=10250" by default instead of "" (and using the default value from the code)? It seems like we should only bother passing the extra parameter when we need to override the default value.